packages/vc-status-list-issuer-rest-api/src/api-functions.ts

```diff
@@ -118,80 +144,91 @@ export function getStatusListCredentialIndexStatusEndpoint(router: Router, conte
         }
       }
       response.statusCode = 200
-      return response.send({ ...entry, status })
+      return response.send({ ...entry, status }) // FIXME content type?
     } catch (e) {
```

Comment:
Do we have a ticket for that or can it be fixed in this ticket?

Reply:
No ticket. Should be in scope of this ticket. The entities already know about the content-type and this is just an if else to set the header

Reply:
fixed